src/components/TradeWidget/index.tsx

```diff
@@ -29,13 +44,28 @@ const SubmitButton = styled.button`
 const TradeWidget: React.FC = () => {
   const { networkId } = useWalletConnection()
   // Avoid displaying an empty list of tokens when the wallet is not connected
-  const fallBackNetworkId = networkId ? networkId : Network.Mainnet
+  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
 
   const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId), [fallBackNetworkId])
-  const [sellToken, setSellToken] = useState(() => getToken('symbol', 'DAI', tokens))
-  const [receiveToken, setReceiveToken] = useState(() => getToken('symbol', 'USDC', tokens))
+
+  // Listen on manual changes to URL search query
+  const urlSeachQuery = useLocation()
+
+  const sellTokenSymbol = urlSeachQuery.get('sell') || 'DAI'
+  const receiveTokenSymbol = urlSeachQuery.get('receive') || 'USDC'
+
+  const [sellToken, setSellToken] = useState(
+    () => getToken('symbol', sellTokenSymbol, tokens) || getToken('symbol', 'DAI', tokens),
```

Comment:
Isn't `else` clause already taken care of three lines above? Or if we want to arbitrarily show DAI/USDC pair for any UNKNOWN/UNKWNOWN pair, shouldn't we rather redirect to DAI/USDC page?